api/src/main/java/org/pmiops/workbench/interactiveanalysis/InteractiveAnalysisService.java

```diff
@@ -56,7 +56,7 @@ public class InteractiveAnalysisService {
   private static final String DATA_URI_PREFIX = "data:application/json;base64,";
   @VisibleForTesting static final String JUPYTER_DELOC_PATTERN = "\\.ipynb$";
 
-  @VisibleForTesting static final String RSTUDIO_DELOC_PATTERN = "\\.(Rmd|R)$";
+  @VisibleForTesting static final String RSTUDIO_DELOC_PATTERN = "(?i)\\.(Rmd|R)$";
```

Comment:
To help me understand the scope of this change: it looks like this is intended to *only* allow .r and .rmd to be case-insensitive, but we will still enforce exactness on .ipynb and .sas?

Reply:
OK, I think this is reasonable.  R really is the weird standout with the capital letter here.